app/src/components/pages/Register.tsx

```diff
@@ -3,6 +3,11 @@ import Button from '@mui/material/Button';
 import TextField from '@mui/material/TextField';
 import Box from '@mui/material/Box';
 import { Grid, Typography } from '@mui/material';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { Faculty } from 'components/shared/constants/timetable.constants';
 
 interface IRegisterForm {

```

Comment:
Update registration form interface, replace vnum with faculty, same goes for the formData object on line 21 - we need to remove vnum from the component and replace it with the faculty dropdown value.